src/bindings/python/CMakeLists.txt

```diff
@@ -9,22 +9,44 @@ if (NOT TARGET pybind11::pybind11_headers)
   add_subdirectory(${VALHALLA_SOURCE_DIR}/third_party/pybind11 ${CMAKE_BINARY_DIR}/third_party/pybind11)
 endif()
 
-pybind11_add_module(_valhalla valhalla/_valhalla.cc)
-target_link_libraries(_valhalla PUBLIC valhalla)
+message(STATUS "Installing python modules to ${Python_SITEARCH}")
```

Comment:
it's pretty annoying that we have two build systems for the bindings. I remember way back in the days when I first published the bindings, I was toying around with a setuptools build extension for cmake, but I also remember very clearly how much of a pain that was. if this ever grows to smth more sizable, we could consider trying it. python packaging is so fragmented and fickle I'd only touch that again if really necessary..